__tests__/app.test.js

```diff
@@ -120,4 +120,46 @@ describe("GET /api/articles/:article_id", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
-  });
\ No newline at end of file
+  });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Responds with an array of comments for the given article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments") 
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toHaveLength(comments.length); 
```

Comment:
same feedback here as when you checked the articles length in the same way - this assertion can never fail!